modules/javafx.media/src/main/native/jfxmedia/jni/JavaInputStreamCallbacks.cpp

```diff
@@ -238,10 +239,9 @@ bool CJavaInputStreamCallbacks::IsSeekable()
         jobject connection = pEnv->NewLocalRef(m_ConnectionHolder);
         if (connection) {
             result = (pEnv->CallBooleanMethod(connection, m_IsSeekableMID) == JNI_TRUE);
+            javaEnv.clearException();
             pEnv->DeleteLocalRef(connection);
         }
-
-        javaEnv.reportException();
```

Comment:
Yes, it should be safe. I do not see any need to print exception, since it is unlikely will happen. Also, these methods called many times to read data and `clearException()` is more light weight. I updated `NeedBuffer()` to use `clearException()`.